src/components/PlantsV2/components/NumberOfSpeciesPlantedCard.tsx

```diff
@@ -18,6 +21,14 @@ export default function NumberOfSpeciesPlantedCard({ plantingSiteId }: NumberOfS
   const [totalSpecies, setTotalSpecies] = useState<number>();
   const [labels, setLabels] = useState<string[]>();
   const [values, setValues] = useState<number[]>();
+  const user = useUser().user;
+  const numberFormatter = useNumberFormatter();
+  const supportedLocales = useSupportedLocales();
+  const numericFormatter = useMemo(
+    () => numberFormatter(user?.locale, supportedLocales),
+    [user?.locale, numberFormatter, supportedLocales]
```

Comment:
suggestion, we should probably expose a wrapper that does this. ideally we don't want to pass supportedLocales everywhere we use numberFormatter. I think useNumberFormatter should handle the supportedLocales automatically